gn3/computations/biweight.py

```diff
@@ -0,0 +1,26 @@
+
+
+"""module contains script to call biweight mid\
+correlation in R"""
```

Comment:
**nitpick:** No need for the extra 2 white spaces at the top of this file. Also, with docstrings, you don't need a "\" to break to a newline :smile_cat: